### ISSUE TITLE
Service generator updates module even when dry run specified

### ISSUE THREAD

**Issue by andrewconnell** [angular/angular-cli#6127]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
@angular/cli: 1.0.1
node: 7.7.3
os: darwin x64

### Repro steps.
```shell
# create project
ng new helloWorld
# create module
ng g m subModule
# create service within module but with --dry-run flag
ng g s subModule/universe -m subModule/subModule.module -spec false -d
```

### Desired functionality.
Dry run is specified... the service is not created, but the module is updated to import & add the service to the `NgModule` decorator like it was added.

**Comment by cexbrayat** [comment]
This should have been fixed in 1.0.1 by https://github.com/angular/angular-cli/commit/be7c9b1b8a1950d92437baf1b36d8a951e2e87dc

I am not reproducing using 1.0.1, do you still have the problem ?

**Comment by andrewconnell** [comment]
I'm certain it happened @cexbrayat , but I can no longer reproduce it so I'll close it.
